graphql-codegen/generated/api.ts

```diff
@@ -1204,6 +1162,83 @@ export type TripsCollectionQuery = {
   } | null
 }
 
+export const GetUserDocument = gql`
+  query getUser($uuid: UUID!) {
+    __typename
+    usersCollection(filter: { uuid: { eq: $uuid } }) {
+      __typename
+      edges {
+        __typename
+        node {
+          __typename
+          uuid
```

Comment:
Wow, Supabase wouldn't generate the query which returns array even though it returns only one user data or null. 
This `edges` looks meaningless though. lol